projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/group-skipping.core-e2e-spec-flaky.cy.ts

```diff
@@ -36,6 +36,7 @@ context('Group Skipping - Checkout', () => {
     cy.requireLoggedIn().then(() => {
       checkout.goToProductDetailsPage();
       checkout.addProductToCart();
+      cy.findByText(/proceed to checkout/i).click();
```

Comment:
This was the only place where `checkout.addProductToCart()` was called. So I moved the action to click `proceed to checkout` after the function call